plugins/plugin-tests/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "plugin-tests"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+rust-version = { workspace = true }
+description = "Tests for the Forc Index plugins"
+
+[dependencies]
+
+forc-index = { workspace = true }
+fuel-indexer-plugin = { version = "0.5" }
```

Comment:
I'm guessing we can't use the `workspace` version of `fuel-indexer-plugin` here?

Reply:
hmm .. we can!